### PR TITLE
(DOCS-15459): Change Stream History Lost Trigger Errors

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Jira

- https://jira.mongodb.org/browse/DOCS-15459

### Staged Changes (Requires MongoDB Corp SSO)

- [Known Issues and Workarounds](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-15459/reference/issues-and-workarounds/#known-issues—workarounds) (and minor changes to linked section, which already mentions "resume a suspended trigger" example and explains the reason for the error)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by dacharyc — APPROVED** [review]
There seems to be something going on with the manual link, but otherwise aside from a small grammar nit LGTM.

File: source/reference/issues-and-workarounds.txt [hunk 1]
```diff
@@ -37,3 +37,27 @@ this issue, there are two possible workarounds:
...
+state due to an error titled . When a ``ChangeStreamHistoryLost`` error
+occurs, the following description appears in the App logs:
+
+.. code-block::
+   :copyable: false
+
+   Resume of change stream was not possible, as the resume
+   point may no longer be in the oplog.
+
+You can :manual:`restart <changeStreams/#resume-tokens>` the Trigger, but
```
[the remark this answers is not in the document]
Comment (nathan-contino-mongo) [reply source/reference/issues-and-workarounds.txt:57]:
```suggestion
You can :manual:`restart </changeStreams/#resume-tokens>` the Trigger, but
```

File: source/triggers/database-triggers.txt [hunk 2]
```diff
@@ -329,25 +329,26 @@ can suspend a Trigger include:
   such as ``dropDatabase``, ``renameCollection``, or those caused by
   a network disruption
 - the token required to resume the change stream is no longer in the
-  cluster :manual:`oplog </core/replica-set-oplog/>`
+  cluster :manual:`oplog </core/replica-set-oplog/>`, called a
+  ``ChangeStreamHistoryLost`` error in the App logs
```
Comment (nathan-contino-mongo) [inline source/triggers/database-triggers.txt:333]:
```suggestion
  cluster :manual:`oplog </core/replica-set-oplog/>`. The App logs
  refer to this as a ``ChangeStreamHistoryLost`` error.

```

File: source/triggers/database-triggers.txt [hunk 3]
```diff
@@ -329,25 +329,26 @@ can suspend a Trigger include:
   such as ``dropDatabase``, ``renameCollection``, or those caused by
   a network disruption
 - the token required to resume the change stream is no longer in the
-  cluster :manual:`oplog </core/replica-set-oplog/>`
+  cluster :manual:`oplog </core/replica-set-oplog/>`, called a
```
[the remark this answers is not in the document]
Comment (nathan-contino-mongo) [reply source/triggers/database-triggers.txt]: Fixed!

**Pull request merged**